pages/api/optimus-metrics.js

```diff
@@ -0,0 +1,41 @@
+import { Client } from '@gradio/client';
+
+export default async function handler(req, res) {
+  if (req.method !== 'GET') {
+    return res.status(405).json({ message: 'Method not allowed' });
+  }
+
+  res.setHeader(
+    'Cache-Control',
+    'public, s-maxage=86400, stale-while-revalidate=43200',
+  );
+
+  try {
+    const client = await Client.connect('valory/Modius-Agent-Performance');
+
+    const result = await client.predict('/refresh_graph', {});
+    const plotString = result.data[0].plot;
+    const plotData = JSON.parse(plotString);
+
+    const traces = plotData.data;
+    const totalTraces = traces.length;
+
+    if (totalTraces < 2) {
+      return res.status(404).json({ error: 'Not enough data traces found' });
+    }
+
+    const ETHAdjustedAPR = traces[totalTraces - 1]?.y;
+    const avgAPR = traces[totalTraces - 2]?.y;
```

Comment:
it's hard to understand what are 1 and 2 (maybe could add some comments), and for totalTraces < 2 maybe could have some const like `MIN_TOTAL_TRACES = 2` for better redability